content/concepts/interoperability/cross-chain-bridges.md

```diff
@@ -0,0 +1,127 @@
+
+
+## Transactions
+
+### Bridge Control: XChainModifyBridge
+
+This transaction modifies a Bridge ledger object on one of the chains that the bridge connects. You can only change the SignaturesReward or MinAccountCreateAmount values, because changing the bridge itself (either door account or either currency) would essentially render all cross-chain funds useless - you’d be better off creating another bridge instead. This transaction must be sent by the door account, and correctly signed using whatever signer list set it has.
+
+### Cross-Chain Transfer: XChainCreateClaimID
+
+This transaction checks out a cross-chain claim ID that is used for a cross-chain transfer. It is submitted on the destination chain, not the source chain. This is the first step of a cross-chain transfer of value. 
+
+A cross-chain claim ID essentially represents one cross-chain transfer of value. 
+
+### Cross-Chain Transfer: XChainCommit
+
+This transaction initiates a cross-chain transfer of value. This is done on the source chain, and locks/burns the value (“commits” the value), so that the equivalent amount can be minted/unlocked on the destination chain. Essentially, it tells the witness servers that the value was locked/burned. This value is tied to a specific cross-chain claim ID (which is included in the transaction).
+
+The account that owns the cross-chain claim ID on the destination chain is the account that controls the funds on the other end of the bridge. The funds go to the destination account specified in the XChainCommit transaction, if specified. If the destination account is not specified, then the claim ID owner must submit an XChainClaim transaction to determine where the funds will go on the destination chain.
+
+### Cross-Chain Transfer: XChainAddAttestation
+
+This transaction is submitted on the destination chain, by the witness server or anyone with access to the signatures from the witness server. It is a proof that an event (essentially just a locking/burning of funds) happened on the source chain.
+
+When enough witnesses have submitted their proofs on the destination chain that an event has occurred, the funds will be released to the destination account in the XChainCommit transaction, if specified. Otherwise, the claim ID owner must submit an XChainClaim transaction to determine where the funds will go on the destination chain.
+
+### Cross-Chain Transfer: XChainClaim
+
+This transaction completes a cross-chain transfer of value. It allows a user to claim the value on the destination chain - the equivalent of the value locked on the source chain. A user can only claim the value if they own the cross-chain claim ID associated with the value locked on the source chain (the Account field). The user can send the funds to anyone (the Destination field). This transaction is only needed if an OtherChainDestination is not specified in the XChainCommit transaction, or if something goes wrong with the automatic transfer of funds.
```

Comment:
Actual emdash. Also, maybe continue to use "assets" rather than changing terms to "value" at this point?

```suggestion
This transaction completes a cross-chain transfer. It allows a user to claim the assets on the destination chain—the equivalent of the assets locked on the source chain. The user must own the cross-chain claim ID associated with the assets locked on the source chain (the `Account` field). The user can send the funds to anyone (the `Destination` field). This transaction is only needed if an OtherChainDestination is not specified in the XChainCommit transaction, or if something goes wrong with the automatic transfer of funds.
```

P.S. the `Account` is always the sender of the transaction; does this mean that the `Account` and `Destination` fields referenced here are on different transactions (one on the source chain, one on the destination chain)? It feels like this sentence says that the cross-chain claim ID is in the `Account` field, but that field should contain an account address, not a unique-per-transfer ID.